index.md

```diff
@@ -11,26 +11,69 @@ layout: default
 
```

Comment:
Have you, by any chance, missed to include a corresponding change to a `index.html` file in a commit?